README.md

```diff
@@ -3,6 +3,10 @@ Fetch Rewards Coding Exercise - Software Engineering - Mobile
 
 Build and run details will be added here, along with ongoing decisions and development notes.
 
+## Development Notes
+- To allow for safer testing and development, the [Item] object returned by the API is wrapped in a Result object.
```

Comment:
The Item object should be fine as is, but we will add a Grouping layer to better handle the `listId` association and display.